Can't compile on 32-bit

E.g. I was cross compiling for armv7-unknown-linux-gnueabihf.

```
error[E0284]: type annotations required: cannot resolve `<f64 as conv::ApproxFrom<usize, _>>::Err == _`
   --> src/quantile.rs:138:29
    |
138 |         let desired_index = f64::approx_from(len).unwrap() * self.p() - 1.;
    |                             ^^^^^^^^^^^^^^^^
```

As far as I can tell, this is since usize is really a u32 on 32-bit platforms, which has the Wrapping ApproxScheme in addition to DefaultApprox.

--- a/src/quantile.rs
+++ b/src/quantile.rs
@@ -1,7 +1,7 @@
 use core;
 use core::cmp::min;
 
-use conv::{ApproxFrom, ConvAsUtil, ValueFrom};
+use conv::{ApproxFrom, ConvAsUtil, ConvUtil, ValueFrom};
 use quickersort::sort_floats;
 
 use super::Estimate;
@@ -81,7 +81,7 @@ impl Quantile {
         ];
         let len = usize::value_from(self.len()).unwrap();  // < 5
         sort_floats(&mut heights[..len]);
-        let desired_index = f64::approx_from(len).unwrap() * self.p() - 1.;
+        let desired_index = ConvUtil::approx_as::<f64>(len).unwrap() * self.p() - 1.;
         let mut index = desired_index.ceil();
         if desired_index == index && index >= 0. {
             let index: usize = index.approx().unwrap();  // < 5

